client/landing/subscriptions/tabs-switcher/tabs-switcher.tsx

```diff
@@ -18,8 +18,8 @@ type TabsSwitcherProps = {
 
 const getRoute = ( baseRoute: string, path: string ): string => {
 	// This is a temporary redirect to the re-skinned Subscription Management portal.
-	// Once the new "Commnets" and "Sites" views are ready, this will be removed.
-	const temporaryRedirect = [ 'comments', 'sites' ];
+	// Once the new "Commnets" views is ready, this will be removed.
```

Comment:
Just noticed one minor insignificant typo:
```suggestion
	// Once the new "Commnets" view is ready, this will be removed.
```